index.html

```diff
@@ -0,0 +1,68 @@
+            </div>
+            <div class="content-container">
+                <h2 class="content-title">Title One</h2>
+                <p class="subheading">posted on 29 June //Category</p>
+                <p class="description">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quibusdam vitae sit,
+                    reiciendis excepturi
+                    maiores quam rem dolorum ea nulla aut laborum placeat facere ullam odit qui est non deserunt
+                    doloribus?
+                </p>
+                <button id="continue-btn">Continue Reading</button>
+            </div>
+        </div>
+        <hr> -->  
```

Comment:
@Kritika263  
According to good coding practices, always remove comment before pushing the code.
So, remove all the commented codes.